integrations/elasticsearch/src/haystack_integrations/document_stores/elasticsearch/document_store.py

```diff
@@ -38,13 +40,13 @@ class ElasticsearchDocumentStore:
 
     Usage example (Elastic Cloud):
     ```python
-    from haystack.document_store.elasticsearch import ElasticsearchDocumentStore
-    document_store = ElasticsearchDocumentStore(cloud_id="YOUR_CLOUD_ID", api_key="YOUR_API_KEY")
+    from haystack_integrations.document_stores.elasticsearch import ElasticsearchDocumentStore
+    document_store = ElasticsearchDocumentStore(api_key_id="YOUR_CLOUD_ID", api_key="YOUR_API_KEY")
```

Comment:
String arguments would no longer work here right?
```suggestion
    document_store = ElasticsearchDocumentStore(
        api_key_id=Secret.from_env_var("ELASTIC_API_KEY_ID", strict=False),
        api_key=Secret.from_env_var("ELASTIC_API_KEY", strict=False),
    )
```

Reply:
oh right!

Reply:
Probably worth listing this PR as a breaking change then I think. Not a big deal, but good to do and perhaps releasing a new major version of the package.

Reply:
sounds good! I think the exclamation point in the title is meant to come after the tag so `feat!:` at least from what I can see in other older PRs

Reply:
upps - corrected it